database/seeds/OrganizationPermissionTypeSeeder.php

```diff
@@ -80,7 +80,13 @@ public function run()
 
         DB::table('organization_permission_types')->insert([
             'name' => 'organization:upload-thumb',
-            'display_name' => 'upload Thumb',
+            'display_name' => 'Upload Thumb',
+            'feature' => 'Organization'
+        ]);
+
+        DB::table('organization_permission_types')->insert([
+            'name' => 'organization:add-role',
```

Comment:
add-role shouldn't  by add_role ?

Reply:
"organization:add-role" is the name of the permission,
we are following these naming conventions for permissions.